gravitee-apim-rest-api/gravitee-apim-rest-api-model/src/main/java/io/gravitee/rest/api/model/UpdateApplicationEntity.java

```diff
@@ -38,38 +42,56 @@
 @AllArgsConstructor
 @Builder
 @JsonIgnoreProperties(ignoreUnknown = true)
+@ToString(onlyExplicitlyIncluded = true)
 public class UpdateApplicationEntity {
 
+    @Getter
     @NotNull(message = "Application's name must not be null")
     @NotEmpty(message = "Application's name must not be empty")
     @Schema(description = "Application's name. Duplicate names can exists.", example = "My App")
+    @ToString.Include
```

Comment:
- can you group lombok annotation to differentiate from validation / swagger ones ?
- only my opinion, but I would have put `@Getter` and `@Setter` annotation at class level and only put an explicit `@Getter(AccessLevel.NONE)`, it is more explicit and easier to maintain imho